managed/src/main/java/com/yugabyte/yw/models/Backup.java

```diff
@@ -225,4 +229,45 @@ public static boolean existsStorageConfig(UUID customerConfigUUID) {
         .collect(Collectors.toList());
     return backupList.size() != 0;
   }
+
+  public static Set<Universe> getUniverses(UUID customerConfigUUID) {
+    List<Backup> backupList = find.query().where()
+        .or()
+          .eq("state", BackupState.Completed)
+          .eq("state", BackupState.InProgress)
+        .endOr()
+        .findList();
+    backupList = backupList.stream()
+        .filter(b -> b.getBackupInfo().storageConfigUUID.equals(customerConfigUUID))
+        .collect(Collectors.toList());
+    Set<UUID> universeUUIDs = new HashSet<>();
+    backupList.stream()
+        .filter(b -> b.getBackupInfo().storageConfigUUID.equals(customerConfigUUID) && 
+        universeUUIDs.add( b.getBackupInfo().universeUUID));   
+        
+    List<Schedule> scheduleList = Schedule.find.query().where()
+        .or()
+          .eq("task_type", TaskType.BackupUniverse)
+          .eq("task_type", TaskType.MultiTableBackup)
+        .endOr()
+        .eq("status", "Active")
+        .findList();
+    scheduleList = scheduleList.stream()
+        .filter(s -> s.getTaskParams().path("storageConfigUUID").asText()
+        .equals(customerConfigUUID.toString()) &&
+        universeUUIDs.add(UUID.fromString(s.getTaskParams().path("universeUUID").toString())))
+        .collect(Collectors.toList());
+    Set<Universe> universes = new HashSet<Universe>();
+    for (UUID universeUUID : universeUUIDs) {
+      try {
```

Comment:
It could be simpler (the item is added only when there is no exception):

`try {
  universes.add(Universe.get(universeUUID));
} catch (Exception e) {
}`

Reply:
Updated

Reply:
You don't need for `continue` in the catch block (you don't have any code after this try/catch).